third_party/proto/cosmos-proto/cosmos.proto

```diff
@@ -7,4 +7,10 @@ option go_package = "github.com/regen-network/cosmos-proto";
 
 extend google.protobuf.MessageOptions {
     string interface_type = 93001;
+
+    string implements_interface = 93002;
+}
+
+extend google.protobuf.FieldOptions {
+    string accepts_interface = 93001;
```

Comment:
@aaronc shall we use a different range here, 94xxx?
```suggestion
    string accepts_interface = 94001;
```

Reply:
Maybe, but this should be coordinated upstream in the cosmos-proto repo